app/core/types/document/errors.go

```diff
@@ -0,0 +1,9 @@
+package document
+
+import "errors"
+
+var (
+
+	// ErrInvalidDocument occurs when a CPF provided by the client is invalid or contains special characters
```

Comment:
I believe the error message you provided is clear enough... the comment would not be adding anything to it :)

Reply:
Thanks for your comment :) I usually write these comments in order to keep a standard among all exported properties.